api/envoy/config/filter/http/transformation/v2/transformation_filter.proto

```diff
@@ -255,6 +255,11 @@ message TransformationTemplate {
   }
   // Use this field to set Dynamic Metadata.
   repeated DynamicMetadataValue dynamic_metadata_values = 9;
+
+  // Use to render the output of a body transformation as JSON. This will cause
+  // rendered string values to be escaped in order to make valid JSON strings
+  bool render_body_as_json = 12;
```

Comment:
i think we should rename this. perhaps this has similar utility for folks who want the body rendered for, say, yaml

Reply:
say `escape_characters`

Reply:
I like it. I renamed the field